### PR TITLE
Update README for 22.08 release

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  be loaded without explicit config.pbtxt file if they are named in form 
+  `<model_name>.<backend_name>`.
+
+* Users can specify a maximum memory limit when loading models onto the GPU  
+  with the new 
+  [--model-load-gpu-limit](https://github.com/triton-inference-server/server/blob/b3d7a3375e7adb1341724c0ac34661b4cde23cd2/src/main.cc#L629-L635)
+  tritonserver option and the 
+  [TRITONSERVER_ServerOptionsSetModelLoadDeviceLimit](https://github.com/triton-inference-server/core/blob/c9cd6630ecb04bb26e2110cd65a37f23aec8153b/include/triton/core/tritonserver.h#L1861-L1872) C API function
+
+* Aded new documentation,
```
Comment (nv-kmcgill53) [inline RELEASE.md:47]: >Aded

Should be Added

**Review by nv-kmcgill53 — CHANGES_REQUESTED** [review]

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+# Release Notes for 2.25.0
+
+## New Freatures and Improvements
+
+* New 
+  [support for multiple cloud credentials](https://github.com/triton-inference-server/server/blob/main/docs/model_repository.md#cloud-storage-with-credential-file-beta) 
+  has been enabled. This feature is in beta and is subject to change.
+
+* Models using custom backends which implement auto-complete configuration, can
```
Comment (nv-kmcgill53) [inline RELEASE.md:37]: nit: remove comma in  "auto-complete configuration, can"

**Review by nv-kmcgill53 — CHANGES_REQUESTED** [review]
Missed this in my first review. This should be the last change.

File: RELEASE.md (hunk 2 above)
Comment (nv-kmcgill53) [inline RELEASE.md:37]: Add link to `auto-complete configuration`: https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration

For copy paste: 
> [auto-complete configuration](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration)

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  In this case the user should provide the full model configuration for these 
+  models or use the `--disable-auto-complete-config` CLI option to show which 
+  models fail to load.
+
+* Auto-complete does not support PyTorch models due to lack of metadata in the 
+  model. It can only verify that the number of inputs and the input names 
+  matches what is specified in the model configuration. There is no model 
+  metadata about the number of outputs and datatypes. Related PyTorch bug: 
+  https://github.com/pytorch/pytorch/issues/38273
+
```
Comment (nv-kmcgill53) [inline RELEASE.md:88]: Add new bullet point as documentation updated:

>Auto-complete is not supported in the OpenVINO backend

**Review by nv-kmcgill53 — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 3 above)
Comment (mc-nv) [reply RELEASE.md:89]: Should be resolved - see below

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+# Release Notes for 2.25.0
+
+## New Freatures and Improvements
+
+* New 
+  [support for multiple cloud credentials](https://github.com/triton-inference-server/server/blob/main/docs/model_repository.md#cloud-storage-with-credential-file-beta) 
+  has been enabled. This feature is in beta and is subject to change.
+
+* Models using custom backends which implement 
+  [auto-complete configuration](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration),
```
Comment (nv-kmcgill53) [inline RELEASE.md:38]: nit: remove comma at end of line

**Review by nv-kmcgill53 — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 4 above)
Comment (mc-nv) [reply RELEASE.md:38]: @nv-kmcgill53 , I copy-paste it from the documentation and the comma present. 
If you assume it's redundant please suggest it in official documentation as well.

**Review by nv-kmcgill53 — APPROVED** [review]

**Review by rmccorm4 — APPROVED** [review]

**Pull request merged**